amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3919,100 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventPaging:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventEntry'
+  EventEntry:
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Event'
+  Event:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+        description: this is the id of the event
+      name:
+        type: string
+        description: This is the unique display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyCreate:
+    type: object
+    required:
+      - name
+      - type
+    properties:
+      name:
+        type: string
+        description: This is the unique display label of the event
+      type:
+        type: string
+        description: this is event type
+        default: Simple
+  EventBodyUpdate:
+    type: object
+    properties:
```

Comment:
I think id and name are required fields here.

Reply:
id is already mandatory field as it's a path param. while updating the object, name or type can be either present so these are provided as optional.

Reply:
Done